src/t2iapi/activation_state/activation_state_requests.proto

```diff
@@ -34,6 +34,19 @@ message SetAlertActivationRequest {
     AlertActivation activation = 2;  // value to set as the ActivationState value
 }
 
+/*
+Request to set the ActivationState of the component or metric with the given handle and the start and
+end @ActivationState value.
+ */
+message SetTransitionComponentActivationRequest {
```

Comment:
Set and Transition doesnt make sense imo
i think "Set" can be dropped from manipulation name, also from the rpc call